eng/pipelines/coreclr/perf.yml

```diff
@@ -18,7 +18,25 @@ trigger:
     - SECURITY.md
     - THIRD-PARTY-NOTICES.TXT
   
-pr: none
+# TODO: add back PR run for testing, will remove
```

Comment:
We need to make sure that this gets removed before we checkin

Reply:
Yes, will do before merge